distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection.enso

```diff
@@ -437,7 +433,9 @@ default_storage_type storage_type = case storage_type of
     Storage.Integer -> Sql_Type.integer
     Storage.Decimal -> Sql_Type.double
     Storage.Boolean -> Sql_Type.boolean
-    Storage.Any -> Sql_Type.blob
+    ## Support for mixed type columns in Table upload is currently very limited,
```

Comment:
Could potentially link it to some improvement ticket here?

Reply:
The `upload` utility is currently an internal one, just for use in testing.

We will be working on a proper 'public' upload component later on - then it may be revisited, but there is no particular ticket for this yet.